src/source/Sctp/Sctp.c

```diff
@@ -183,6 +183,26 @@ STATUS sctpSessionWriteMessage(PSctpSession pSctpSession, UINT32 streamId, BOOL
     return retStatus;
 }
 
+//https://tools.ietf.org/html/draft-ietf-rtcweb-data-protocol-09#section-5.1
+//      0                   1                   2                   3
+//      0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+//     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+//     |  Message Type |  Channel Type |            Priority           |
+//     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+//     |                    Reliability Parameter                      |
+//     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+//     |         Label Length          |       Protocol Length         |
+//     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+//     \                                                               /
+//     |                             Label                             |
+//     /                                                               \
+//     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+//     \                                                               /
+//     |                            Protocol                           |
+//     /                                                               \
+//     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+#define DCEP_LABEL_LEN_OFFSET 8
+#define DCEP_LABEL_OFFSET 12
```

Comment:
NICE!!! Do you mind moving the declarations to the Sctp.h file? So sorry to pester you with these.

Reply:
i thought these were too implementation specific to move to header.
why would any other piece of code want to know about the packet-internal offsets

Reply:
Indeed. This is why I suggested Sctp.h which has Sctp specifics. We try to not use definitions in the C files.

Reply:
done